doc/schemas/lightning-bkpr-editdescriptionbypaymentid.json

```diff
@@ -0,0 +1,264 @@
+                "properties": {
+                  "type": {
+                    "type": "string",
+                    "enum": [
+                      "onchain_fee"
+                    ]
+                  }
+                }
+              },
+              "then": {
+                "properties": {
+                  "account": {},
+                  "type": {},
+                  "tag": {},
+                  "credit_msat": {},
+                  "debit_msat": {},
+                  "currency": {},
+                  "timestamp": {},
+                  "description": {},
+                  "txid": {
+                    "type": "txid",
+                    "description": [
+                      "The txid of the transaction that created this event."
+                    ]
+                  }
+                },
+                "required": [
+                  "txid"
+                ],
+                "additionalProperties": false
+              }
+            },
+            {
+              "if": {
+                "properties": {
+                  "type": {
+                    "type": "string",
+                    "enum": [
+                      "channel"
+                    ]
+                  }
+                }
+              },
+              "then": {
+                "properties": {
+                  "account": {},
+                  "type": {},
+                  "tag": {},
+                  "credit_msat": {},
+                  "debit_msat": {},
+                  "currency": {},
+                  "timestamp": {},
+                  "description": {},
+                  "fees_msat": {
+                    "type": "msat",
+                    "description": [
+                      "Amount paid in fees."
+                    ]
+                  },
+                  "is_rebalance": {
+                    "type": "boolean",
+                    "description": [
+                      "Is this payment part of a rebalance."
+                    ]
+                  },
+                  "payment_id": {
+                    "type": "hex",
+                    "description": [
+                      "Lightning payment identifier. For an htlc, this will be the preimage."
+                    ]
+                  },
+                  "part_id": {
+                    "type": "u32",
+                    "description": [
+                      "Counter for multi-part payments."
+                    ]
+                  }
+                },
+                "additionalProperties": false
+              }
+            }
+          ]
+        }
+      }
+    }
+  },
+  "author": [
+    "Lisa Neigut <<niftynei@gmail.com>> is mainly responsible."
+  ],
+  "see_also": [
+    "lightning-bkpr-editdescriptionbyoutpoint(7)",
+    "lightning-bkpr-listaccountevents(7)",
+    "lightning-bkpr-listincome(7)"
+  ],
+  "resources": [
+    "Main web site: <https://github.com/ElementsProject/lightning>"
+  ],
+  "examples": [
```

Comment:
did we want examples autogenerated for these?

Reply:
Yes! Do you know how to do that?

Reply:
Been trying since yesterday but it keeps erroring out, ill keep trying i need to debug more closely

Reply:
very close now

Reply:
added in https://github.com/niftynei/lightning/pull/9